<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WirelessSettingCopyRouterSettingPage</class>
 <widget class="QWidget" name="WirelessSettingCopyRouterSettingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">WirelessSettingCopyRouterSettingPage{background-color:transparent}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="m_copyGB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Wireless Setting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QLabel" name="m_routerFounedLB">
        <property name="text">
         <string>The Wireless Setup Wizard found a NETGEAR router with below settings,You can apply the same settings to the XAVN2001 to make it a wireless extender for your router,or you can change the setting to create a separate network.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="m_routerGB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Your Router's Wireless Settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="WirelessSettingReadonlyPannel" name="m_routerSettingPane" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="m_plcGB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>XAVN2001's Wireless Settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="WirelessSettingPannel" name="m_wirelessSettingPane" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="WirelessSettingOperatePane" name="m_operatePane" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WirelessSettingPannel</class>
   <extends>QWidget</extends>
   <header>wirelesssettingpannel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WirelessSettingOperatePane</class>
   <extends>QWidget</extends>
   <header>wirelesssettingoperatepane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WirelessSettingReadonlyPannel</class>
   <extends>QWidget</extends>
   <header>wirelesssettingreadonlypannel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
